release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -110,6 +110,16 @@ The metric names that start with `ovn` are unique to the OVN-Kubernetes CNI netw
 
 The `ovnkube_master_resource_update_total` metric is removed for the 4.10 release.
 
+[id="ocp-4-10-networking-must-gather-tcpdump"]
+==== Enhancement to must-gather to simplify network tracing
+
+The `oc adm must-gather` command is enhanced to simplify collecting network packet captures.
+You can collect packet captures on multiple nodes at the same time.
+A new `--node-selector` argument provides a way to identify the nodes for the packet captures.
```

Comment:
"for the packet captures"
I found this a bit unclear. Perhaps something like thi? "to identify which nodes you're collecting packet captures for."

Reply:
This note might be more helpful if you mention explicitly what's changed. Something like: It used to be that you could only do this on one node at a time, and now you can do bazillions!